### DISCUSSION TITLE
Allow export of specific (selected) items

### DISCUSSION THREAD

**Discussion by yarick2000** [directus/directus#5898] [Feature Requests]
I would like to have the following functionality  implemented:
Right now the export is only possible for the whole collection (or at least those items that currently filtered). I would suggest to extend it by making possible exporting of selected items only. So when user:
a) Has no items selected - the whole view is exported
b) Has one or more items selected - only selected items exported
c) Has all items selected (in Table View) - the whole view is exported

**Comment by azrikahar** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
